conf.py

```diff
@@ -67,10 +67,13 @@
 # List of patterns, relative to source directory, that match files and
 # directories to ignore when looking for source files.
 # This pattern also affects html_static_path and html_extra_path.
-# exclude index.rst files for metapackages for rosdoc2
+# exclude index.rst files from packages/metapackages for rosdoc2
 exclude_patterns = ["_build", "Thumbs.db", ".DS_Store",
                     "**/CHANGELOG.rst", "**/README.rst",
                     "doc/ros2_control/ros2_control/**.rst",
+                    "doc/realtime_tools/doc/**.rst",
+                    "doc/control_msgs/doc/**.rst",
+                    "doc/control_toolbox/doc/**.rst",
```

Comment:
@christophfroehlich What about the package `kinematic_interface`?

Reply:
I haven't added rosdoc2 stuff there yet, but I can add it here.